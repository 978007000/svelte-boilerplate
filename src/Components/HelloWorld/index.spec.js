import HelloWorld from './index.svelte';

describe('Sample', () => {
	it('is empty by default', (next) => {
		const target = document.createElement('div');
		const sample = new HelloWorld({ target });
        console.log(sample);

        setTimeout(() => {
			const { firstElementChild: element } = target;

            expect(element.textContent).toBe('Hello World! Hello World!');

            next();
		}, 10);
	});
});