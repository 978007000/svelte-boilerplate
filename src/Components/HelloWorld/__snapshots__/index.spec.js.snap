// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World renders Hello World! text 1`] = `
<body>
  <div>
    <div
      class="title"
    >
      <span
        class="title__back"
      >
        Hello World!
      </span>
       
      <span
        class="title__front"
      >
        Hello World!
      </span>
    </div>
  </div>
</body>
`;
